src/praisonai-code/praisonai_code/cli/commands/tools.py

```
@@ -18,11 +18,90 @@
+        return tools, unavailable
+
+    # Suppress the handler's own status prints so discovery stays quiet; a
+    # failed connection surfaces as an unavailable row instead.
+    handler = MCPHandler()
+    handler.print_status = lambda *a, **k: None  # type: ignore[assignment]
+
+    for server in servers:
+        name = str(server.get("name") or server.get("command") or server.get("url") or "mcp")
+        try:
+            mcp = handler.create_mcp_from_server(server)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)